src/app/public/modules/file-attachment/file-item.ts

```diff
@@ -3,8 +3,20 @@ import {
 } from './file-item-error-type';
 
 export interface SkyFileItem {
+  /**
+   * The object which has been added or removed.
+   */
   file: File;
+  /**
+   * The data URL for the file which has been added or removed.
```

Comment:
```suggestion
   * The data URL for the file that was added or removed.
```

Also, why do we have `data` before `URL` here but not in the description under `SkyFileLink`? I assume we should delete it here unless it is this adding important information. If so, we should we add `data` to other instances of `URL` in these docs, shouldn't we?

Reply:
Nope. This actually was an intended difference. Data URLs, in a nutshell, are encoded versions of the file.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs

Reply:
Is that common knowledge for developers, or should we make "data URL" a link to the URL you shared?

Also, can we make the suggested tweak to the wording?

Reply:
Oops. Totally missed that suggestion. Done and also added that link - I don't think it hurts anything to be specific there.